src/components/Article.jsx

```diff
@@ -0,0 +1,33 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+import ArticleCard from "./ArticleCard";
+import CommentsList from "./CommentsList";
+import CommentAdder from "./CommentAdder";
+
+function Article () {
+    const {article_id} = useParams();
+    const [articleRequest, setArticleRequest] = useState(`https://jay-mckerracher-nc-news.onrender.com/api/articles/${article_id}`)
+    const [article, setArticle] = useState({})
+    const [isLoading, setIsLoading] = useState(true)
+
+    useEffect(() => {
+        axios
```

Comment:
we should extract this out to its own file in a utils folder - we can then pass the id directly from use params to this function which will request the particular article